Yosemite/Yosemite/PointOfSale/POSProductProvider.swift

```diff
@@ -1,19 +1,19 @@
 import Foundation
-import Yosemite
 import protocol Storage.StorageManagerType
 import class WooFoundation.CurrencySettings
 
 /// Product provider for the Point of Sale feature
 ///
-final class POSProductProvider {
+public final class POSProductProvider: POSItemProvider {
     private let storageManager: StorageManagerType
     private var siteID: Int64
     private var currencySettings: CurrencySettings
 
-    init() {
-        self.storageManager = ServiceLocator.storageManager
-        self.siteID = ServiceLocator.stores.sessionManager.defaultSite?.siteID ?? 0
-        self.currencySettings = ServiceLocator.currencySettings
+    public init(storageManager: StorageManagerType, siteID: Int64, currencySettings: CurrencySettings ) {
+        // ServiceLocator is part of the WooCommerce binary, expose
```

Comment:
Oops I committed this comment by mistake, I'll remove it before merging.

Reply:
Removed on https://github.com/woocommerce/woocommerce-ios/pull/12882/commits/6a4ec649a6b2a40a07a8e735024f6f707e12be0e